packages/api/src/routes/user-did-register.js

```diff
@@ -0,0 +1,24 @@
+import { validate } from '../utils/auth.js'
+import { JSONResponse } from '../utils/json-response.js'
+import { parse } from 'ucan-storage/did'
+
+/** @type {import('../bindings').Handler} */
+export const userDidRegister = async (event, ctx) => {
```

Comment:
nit
`userDidRegister` => `userDIDRegister` 
A bit more readable and avoid having `did` read like a verb.